.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_size = 4
+indent_style = space
```

Comment:
Can we keep tabs for indentation ?

Reply:
> Can we keep tabs for indentation ?

In case anyone missed it see my [comment](https://github.com/att/ast/issues/125#issuecomment-346701165) in the issue that prompted this PR.